index.html

```diff
@@ -30,7 +27,9 @@
   <section class="search-section">
     <input type="text" placeholder="Search software jobs" />
     <button class="search-button">
-      <span class="fa fa-search"></span>
+      <svg class="icon">
+        <use xlink:href="images/icons.svg#icon-search"></use>
```

Comment:
Because it is decoration. Screen readers know it is a submit button regardless of inner HTML (submit attribute was addd in another commit or PR)